gradle/forge.gradle

```diff
@@ -36,7 +39,14 @@ dependencies {
 
     // Project lombok
     provided "org.projectlombok:lombok:1.16.8"
+
     testCompile "junit:junit:4.12"
+    testCompile "com.jjtparadox.barometer:barometer:0.0.3-1.12"
+    testCompile makeStart.outputs.files
+
+    testRuntime "org.jetbrains.kotlin:kotlin-stdlib:1.1.2-5"
+    testRuntime "org.jetbrains.kotlin:kotlin-reflect:1.1.2-5"
+    testRuntime "io.github.lukehutch:fast-classpath-scanner:2.18.1"
```

Comment:
It should be possible to automatically include these deps by declaring them in the pom of barometer.

Reply:
That's my fault, as I still lack experience in publishing to maven repos :pensive: I'll try again at getting my maven repo working.

@tdaffin if you figure out how to get the pom working, please let me know! I'd appreciate the help :smile:

Reply:
I pushed the change to use @jjtParadox 's repository, along with the removal of these now unnecessary lines.